tests/kernel/ibtests.pm

```diff
@@ -127,7 +127,7 @@ sub post_fail_hook {
     my $role = get_required_var('IBTEST_ROLE');
 
     if ($role eq 'IBTEST_MASTER') {
-        script_run('tr -cd \'\11\12\15\40-\176\' < results/TEST-ib-test.xml > /tmp/results.xml');
+        script_run('tr -cd \'\11\12\15\40-\176\' < results/TEST-ib-test.xml > /tmp/results.xml', 0);
```

Comment:
have you forgotten `timeout =>` before 0

Reply:
I removed all timeouts of 0